src/main/java/org/broadinstitute/hellbender/tools/exome/NormalizeSomaticReadCounts.java

```diff
@@ -242,7 +242,7 @@ private void writeTargetFactorNormalizedOutput(final ReadCountCollection targetF
      */
     private void writeTangentNormalizedOutput(final ReadCountCollection tangentNormalized) {
         try {
-            ReadCountCollectionUtils.writeAsTargetCoverage(outFile, tangentNormalized, "fileFormat = tsv",
+            ReadCountCollectionUtils.write(outFile, tangentNormalized, "fileFormat = tsv",
```

Comment:
Are you *sure* this does not break something downstream?  Did you run the evaluation workflow?  Or make sure that the output is exactly the same as it was before?